client/apps/edit/components/section_image_set/index.styl

```diff
@@ -0,0 +1,263 @@
+@import '../stylus_lib'
+
+artwork-img-bottom = 10px
+short-input-width = 290px
+wide-input-width = 640px
+
+.edit-section-image-set
+  position relative
+  margin-top 20px
+
+.edit-section-image-set .edit-section-controls
```

Comment:
Great namespacing/stylus work! Likewise with the line numbers here—not _too_ long yet, but if this thing grows also consider breaking down the stylesheets.